packages/gestalt-codemods/33.0.0/heading-replace-truncate-lineClamp.js

```diff
@@ -70,6 +70,7 @@ export default function transformer(file, api) {
             console.log(
               `${node.openingElement.name.name} components with ${attr?.name?.name} prop must be converted to lineClamp manually (boolean -> number). Location: ${file.path} @line: ${node.loc.start.line}`,
             );
+            return null;
```

Comment:
There was a bug in this codemod that I hadn't looked into before. This fixes it.